keps/sig-network/2943-networkpolicy-status/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: Network Policy status subresource
+kep-number: 2943
+authors:
+  - "@rikatz"
+owning-sig: sig-network
+status: provisional
```

Comment:
why not implementable?

Reply:
changed to implementable. I left this as provisional due to the time limit vs kep freeze, so if we decide to merge but not implement right now the status would be correct. Now it's implementable